index.html

```diff
@@ -42,6 +41,133 @@ <h1><span class="color-blue"> Hey there. <br> I'm Ale. </span><br><span id="titl
+                    <section class="project-lang">
+                        <ul class="project-lang-list">
+                            <li>Ruby on Rails</li>
+                            <li>CSS</li>
+                            <li>JavaScript</li>
+                            <li>HTML</li>
+                        </ul>
+                    </section>
+                    <button>See project</button>
+                </div>   
+            </section>
+            <section class="work n3">
+                <div class="white-area">
+                    <h4>Multi-Post Stories Gain+Glory</h4>
+                    <section class="project-lang">
+                        <ul class="project-lang-list">
+                            <li>Ruby on Rails</li>
+                            <li>CSS</li>
+                            <li>JavaScript</li>
+                            <li>HTML</li>
+                        </ul>
+                    </section>
+                    <button>See project</button>
+                </div>   
+            </section>
+            <section class="work n4">
+                <div class="white-area">
+                    <h4>Multi-Post Stories Gain+Glory</h4>
+                    <section class="project-lang">
+                        <ul class="project-lang-list">
+                            <li>Ruby on Rails</li>
+                            <li>CSS</li>
+                            <li>JavaScript</li>
+                            <li>HTML</li>
+                        </ul>
+                    </section>
+                    <button>See project</button>
+                </div>   
+            </section>
+            <section class="work n5">
+                <div class="white-area">
+                    <h4>Multi-Post Stories Gain+Glory</h4>
+                    <section class="project-lang">
+                        <ul class="project-lang-list">
+                            <li>Ruby on Rails</li>
+                            <li>CSS</li>
+                            <li>JavaScript</li>
+                            <li>HTML</li>
+                        </ul>
+                    </section>
+                    <button>See project</button>
+                </div>   
+            </section>
+            <section class="work n6">
+                <div class="white-area">
+                    <h4>Multi-Post Stories Gain+Glory</h4>
+                    <section class="project-lang">
+                        <ul class="project-lang-list">
+                            <li>Ruby on Rails</li>
+                            <li>CSS</li>
+                            <li>JavaScript</li>
+                            <li>HTML</li>
+                        </ul>
+                    </section>
+                    <button>See project</button>
+                </div>   
+            </section>
+        </section>
+        <section class="about-myself">
+            <div class="text">
+                <h1 class="color-blue">About me</h1>
```

Comment:
- [ ] Following the best practices, you are meant to have only one `h1` tag on your page and that should be the main title, this is not the main title, and therefore ti should be in `2h or h3` tag, I'll suggest you update this accordingly. :+1: :smile: